components/permissions/google_sign_in/google_sign_in_permission_throttle.h

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_PERMISSIONS_GOOGLE_SIGN_IN_GOOGLE_SIGN_IN_PERMISSION_THROTTLE_H_
+#define BRAVE_COMPONENTS_PERMISSIONS_GOOGLE_SIGN_IN_GOOGLE_SIGN_IN_PERMISSION_THROTTLE_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "base/memory/scoped_refptr.h"
+#include "base/memory/weak_ptr.h"
+#include "components/content_settings/core/browser/host_content_settings_map.h"
+#include "content/public/browser/browser_context.h"
+#include "content/public/browser/web_contents.h"
+#include "third_party/blink/public/common/loader/url_loader_throttle.h"
+
+namespace permissions {
+
+class GoogleSignInPermissionThrottle : public blink::URLLoaderThrottle {
```

Comment:
this now should be moved to `google_sign_in_permission` component.

Reply:
Actually I guess it doesn't any more.